pkgs/development/python-modules/pycfdns/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, aiohttp
+, async-timeout
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "pycfdns";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
```

Comment:
Why use `fetchFromGitHub` if there are no tests?

Reply:
I prefer to use the upstream location of the source when ever possible.